upf/upf_pfcp.c

```diff
@@ -72,9 +73,12 @@ format_upf_device_name (u8 * s, va_list * args)
   u32 i = va_arg (*args, u32);
   upf_nwi_t *nwi;
 
-  nwi = pool_elt_at_index (gtm->nwis, i);
+  if (!pool_is_free_index (gtm->nwis, i))
+    {
+      nwi = pool_elt_at_index (gtm->nwis, i);
+      s = format (s, "upf-nwi-%U", format_network_instance, nwi->name);
+    }
```

Comment:
This can be happen if we will delete nwi with configured UPIP and then try device output CLI command
as example - "show hardware"
I will move fixes of nwi deletion in a separate PR